src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -200,7 +200,7 @@ public static boolean checkTree(@NotNull final IBlockAccess world, final BlockPo
                 for (int z = -1; z <= 1; z++)
                 {
                     final BlockPos temp = log.add(x, y, z);
-                    if (world.getBlockState(temp).getBlock().isWood(null, temp) && !woodenBlocks.contains(temp))
+                    if ((world.getBlockState(temp).getBlock().isWood(null, temp) || world.getBlockState(temp).getMaterial() == Material.CLAY ) && !woodenBlocks.contains(temp) && world.getBlockState(temp).getBlock() != Blocks.CLAY)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 230 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)